code/glTF2AssetWriter.inl

```diff
@@ -196,51 +206,158 @@ namespace glTF2 {
     }
 
     namespace {
-        inline void WriteTex(Value& obj, Ref<Texture> texture, const char* propName, MemoryPoolAllocator<>& al)
+        inline void SetTexBasic(TextureInfo t, Value& tex, MemoryPoolAllocator<>& al)
```

Comment:
This method I created to be used in the various forms of `WriteTex`. Is this the right way to create a reusable function that's used only within the `WriteTex` functions?

Reply:
Yeah, that's a good trick!